extensions/hucode-setup-ui/src/styles/setup.css

```
@@ -0,0 +1,210 @@
+
+@source "../components/**/*.tsx";
+@source "../import/**/*.tsx";
+@source "../lib/**/*.ts";
+@source "../vendor/**/*.tsx";
+
+/*
+ * Hucode owns four explicit palettes. The webview infrastructure puts one of its mode classes on
+ * the document body, and that class is the renderer's only workbench theme input: `transformCssVariables`
+ * removes the `--vscode-*` map, so nothing here may reference it.
+ */
+@custom-variant hucode-light (&:where(body.vscode-light, body.vscode-light *));
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@custom-variant`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)